lib/galaxy/tools/actions/upload_common.py

```diff
@@ -376,7 +376,7 @@ def create_paramfile(trans, uploaded_datasets):
             # TODO: This will have to change when we start bundling inputs.
             # Also, in_place above causes the file to be left behind since the
             # user cannot remove it unless the parent directory is writable.
-            if link_data_only == 'copy_files' and trans.user and trans.app.config.external_chown_script:
+            if link_data_only == 'copy_files' and trans.user and trans.app.config.external_chown_script and not (uploaded_dataset.path.startswith("http://") or uploaded_dataset.path.startswith("https://") or uploaded_dataset.path.startswith("ftp://")):
```

Comment:
In https://github.com/galaxyproject/galaxy/blob/dev/tools/data_source/upload.py#L199 we're using
```suggestion
            if link_data_only == 'copy_files' and trans.user and trans.app.config.external_chown_script and uploaded_dataset.path.find('://') == -1:
```
might be worth moving that into galaxy.util for consistency and so that we can use something fancier later ?